### PR TITLE
Some translations missing

### PR DESCRIPTION
**Pull request by Aejoh**

### CODE REVIEW

**Comment by ChristophWurst** [comment]
Hi @Aejoh,
please submit translations via Transifex: https://www.transifex.com/projects/p/owncloud/

For more detailed information about translations: http://doc.owncloud.org/server/9.1/developer_manual/core/translation.html

cc @DeepDiver1975 

Thank you very much :-)

File: l10n/de_DE.js [hunk 1]
```diff
@@ -58,6 +58,14 @@ OC.L10N.register(
...
-    "Continue to %s" : "Fortsetzen auf %s"
+    "Continue to %s" : "Fortsetzen auf %s",
+    "New message" : "Neue Nachricht",
+    "to" : "An",
+    "Subject" : "Betreff",
+    "Message …" : "Nachricht ...",
+    "Forward" : "Weiterleiten",
+    "No messages in this folder!" : "Keine Nachrichten in diesem Ordner",
+    "from" : "Von",
+    "Add attachment from Files" : "Anhänge hinzufügen"
```
Comment (ChristophWurst) [inline l10n/de_DE.js]: I'd say "Anhang" is a better options for this string.

File: l10n/de_DE.js [hunk 2]
```diff
@@ -58,6 +58,14 @@ OC.L10N.register(
     "Redirect warning" : "Umleitungswarnung",
     "The previous page is sending you to %s" : "Die vorangegangene Seite leitet Sie zu %s weiter",
     "If you do not want to visit that page, you can return to <a href=\"%s\">the mail app</a>." : "Wenn Sie diese Seite nicht besuchen möchten, können Sie <a href=\"%s\">zur Mail-App</a> zurück kehren.",
-    "Continue to %s" : "Fortsetzen auf %s"
+    "Continue to %s" : "Fortsetzen auf %s",
+    "New message" : "Neue Nachricht",
+    "to" : "An",
+    "Subject" : "Betreff",
+    "Message …" : "Nachricht ...",
```
Comment (Scheirle) [inline l10n/de_DE.js]: `...` (3 dots) should be replaced with the Unicode [`HORIZONTAL ELLIPSIS`](http://www.fileformat.info/info/unicode/char/2026/index.htm) symbol (as in the original non-translated version).

Comment (ChristophWurst) [inline l10n/de_DE.js]: Tip: you can type that three-dots-char with ``Alt Gr`` + ``.`` on German keyboards ;-)

**Pull request closed without merging**